roadmaps/external-deps.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: contribute
+title: Bazel External Deps Roadmap 
+---
+<style>
+  .padbottom { padding-bottom: 10px; }
+  .etabox {
+    background: #EFEFEF;
+    color: #38761D;
+    font-size: 15px;
+    font-weight: bold;
+    display: inline;
+    padding: 6px;
+    margin-right: 10px;
+  }
+  .donestatus {
+    color: #00D000;
+    font-weight: bold;
+    padding-left: 10px;
+  }
+  .inprogressstatus {
+    color: #D0D000;
+    font-weight: bold;
+    padding-left: 10px;
+  }
+</style>
+
+# Bazel External Deps Roadmap
+
+_Last verified: July 17, 2018 
+([Update history](https://github.com/bazelbuild/bazel-website/commits/master/roadmaps/external-deps.md))_
+
+Roadmap for improving external dependencies support and integration with
+package managers and other build systems in Bazel.
+
+
+External dependencies work fall into three broad categories:
+
+1.  General improvements for WORKSPACE support in Bazel (see 
+   [External Repositories: The Future](https://docs.google.com/document/d/13OlvrQcPG15cnrY5InEwJ4J3rV_ykHTuHqEZVTqY2lk/) 
+   for general background and discussion).
+
+1.  Allowing Bazel project to depend on other projects built with a different build tool and/or
+   available through a package manager.
+   *  [bazel-deps](https://github.com/johnynek/bazel-deps) is one tool that provide that support. 
+   *  [A good analysis](https://blog.envoyproxy.io/external-c-dependency-management-in-bazel-dd37477422f5)
+      of challenges with C++ dependencies has been done by Envoy project.
+
+1. Allowing other build systems to depend on Bazel project.
+   *   Generally this includes features such as 
+       -   plugins for other build systems invoking Bazel
+       -   publishing Bazel-built artifacts so that they can be used by other build systems
+       -   generating project files  for other build systems
+   *  The full set of features needed for this is currently forming. (We welcome your input on bazel-discuss@)
+
+
+## 2018 Q2
```

Comment:
Is it useful to list previous quarters?

Either it should be done (then, we don't really need it on the roadmap), or postponed to the next quarter.

Reply:
I actually think it is useful for people to understand what has already been done as well as what is coming, at least to some extent.

Reply:
What has been done can be:
* announced in the release notes
* described in the documentation

Reply:
We should not optimize for only saying things in one place :)
Why saying things in release notes and documentation precludes saying things here?

Reply:
ok, as long as we don't spend time to write/maintain it, I'm fine with that.